plugins/catalog/src/components/EntitySwitch/EntitySwitch.tsx

```diff
@@ -55,14 +55,14 @@ type SwitchCaseResult = {
 
 /**
  * Props for the {@link EntitySwitch} component.
- * @public
+ * @public @deprecated import from plugin-catalog-react instead
  */
 export interface EntitySwitchProps {
   children: ReactNode;
   renderMultipleMatches?: 'first' | 'all';
 }
 
-/** @public */
+/** @public @deprecated import from plugin-catalog-react instead */
```

Comment:
Another way to potentially handle this is to just re-export these components/functions from `@backstage/plugin-catalog-react` (but in a way where it's still clear via `api-reports.md` and typescript that the import is `deprecated`).

Reply:
Good shout/did this following Patrik's suggestion as well!